src/cloud-functions.ts

```diff
@@ -141,28 +145,33 @@ export namespace Change {
     if (json.fieldMask) {
       before = applyFieldMask(before, json.after, json.fieldMask);
     }
+
     return Change.fromObjects(
       customizer(before || {}),
       customizer(json.after || {})
     );
   }
 
-  /** @hidden */
+  /**
+   * @hidden
```

Comment:
Let's keep these in one line too, applies throughout.

Reply:
85eb8a0af6291d0dc3b0109497a41ba9a458fca4